docs/api.md

```diff
@@ -113,32 +67,49 @@ parameters = {
     "name": "GhsPopComparisonBuildings",
     "layerName": "building_count",
     "dataset": "regions",
-    "featureId": "3",
+    "featureId": 3,
+    "fidField": "ogc_fid",  # Optional
+    "includeSvg": False,  # Optional
+    "includeHtml": False,  # Optional
 }
-response = requests.get(url, params=parameters)
+response = requests.post(url, json=parameters)
 assert response.status_code == 200
```

Comment:
same question

Reply:
Removed the assert statement as it is not necessary.